internal/resolution/manifest/maven.go

```diff
@@ -827,7 +833,8 @@ func writeDependency(w io.Writer, enc *xml.Encoder, raw string, patches map[Mave
 	return enc.Flush()
 }
 
-func writeString(enc *xml.Encoder, raw, tag string, values map[string]string) error {
+// noInnder indicates whether the element of the given XML name only contains string but no more inner element.
```

Comment:
nit: noInner is not spelled correctly. Also, since we are adding a comment here, can we document the function fully as well? 

i.e.

```
// writeString writes ..... 
```